### PR TITLE
Modified Parent_Test and runner_thread

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Parent_test:
============
Modified init and configure

Runner_thread:
==============
Removed the mixin call using RE_MIXIN and modified TC_OBJ as per the changes in Parent_test

Updates: #75

Signed-off-by: aujjwal-redhat <aujjwal@redhat.com>

### CODE REVIEW

File: tests/parent_test.py [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
 
 class ParentTest:
 
     """
     This class contains the standard info and methods which are needed by
     most of the tests
 
-    PASS: states the pass value
-
+    PASS: states the pass value i.e True
```
Comment (srijan-sivakumar) [inline tests/parent_test.py]: nit. We have only only TEST_RES

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]
Just remove the detail about the pass and fail form the class doc.

File: tests/parent_test.py (hunk 1 above)
Comment (srijan-sivakumar) [reply tests/parent_test.py]: Now..you pushed a commit before I could comment :p

**Review by srijan-sivakumar — APPROVED** [review]

File: tests/parent_test.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/parent_test.py]: XD sorry

**Pull request merged**